### ISSUE TITLE
Application overview blank

### ISSUE THREAD

**Issue by bremnes** [Unleash/unleash#340]
Hi,

I'm testing Unleash, but can't seem to understand "applications". What I expected was that my unleashed-enabled applications would register on startup and then appear in the "applications" list with toggle and strategy information, in addition with their instance(s) and last seen time.

I've used [unleash-docker](https://github.com/Unleash/unleash-docker), a custom .net core web app with the [unleash-client-dotnet](https://github.com/stiano/unleash-client-dotnet) and [unleash-demo-app-node](https://github.com/Unleash/unleash-demo-app-node/), but nothing shows up. If I look in the postgresql database the tables `client_applications` `client_instances` are empty.

Is there any documentation I've missed or somebody who could explain? Thanks!

**Comment by ivarconr** [comment]
This sounds very strange. You expectations is absolutely right, the applications overview is intended to give you an overview of all clients registered. At startup, client SDKs should use the [register endpoint](https://github.com/Unleash/unleash/blob/master/docs/api/client/register-api.md) to do this. 

It might be that there is something odd going on with the dotnet client, which I have personally never used, but the node client is working well for us in over tens of applications in production right now, and all applications seems to be registered correctly. 

Would it be possible for you verify the same behaviour if you using http://unleash.herokuapp.com/api/ in your clients (I have just wiped the demo-db at heroku)? The UI is located at http://unleash.herokuapp.com

**Comment by bremnes** [comment]
Thanks for replying, @ivarconr 

I tried again with the node test app and .net core app. It's working now:
* For the node test app I had only switched out one of the two hard coded unleash server urls. Found the other one as well and then it appeared on the applications page.
* .net core client seems to have a small bug which tricked me with the startup log: `Send metrics interval: newer`. Looking at the source code it's actually supposed to say 'never'. By setting the property `SendMetricsInterval` it works for .net core as well. (I'll report it.)
